Inventor_API/Models/Category.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Inventor_API.Models
+{
+    public class Category
```

Comment:
You can use a base class model to avoid repeating common properties and don't forget the virtual key word